File: 151003/Mytnik/Lab2/Lab2/Lab2/DTO/CreatorRequestTo.cs
using Lab2.Models;
using Lab2.DTO.Interface;

namespace Lab2.DTO
{
    public class CreatorRequestTo(int Id,string? Login, string? Password, string? Firstname, string? Lastname) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public string? Login { get; set; } = Login;

        public string? Password { get; set; } = Password;

        public string? Firstname { get; set; } = Firstname;

        public string? Lastname { get; set; } = Lastname;

      
    }
}

File: 151004/Karvatskaya/lab2/src/main/java/com/example/lab2/DTO/CreatorRequestTo.java
package com.example.lab2.DTO;

import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreatorRequestTo {
    int id;
    @Size(min = 2, max = 64)
    String login;
    String password;
    String firstname;
    String lastname;
}
